routers/api/v1/org/member.go

```diff
@@ -236,18 +236,16 @@ func PublicizeMember(ctx *context.APIContext) {
 	if ctx.Written() {
 		return
 	}
-	allowed := userToPublicize.ID != ctx.Doer.ID && !ctx.Doer.IsAdmin
-	if !allowed {
+	if userToPublicize.ID != ctx.Doer.ID && !ctx.Doer.IsAdmin {
```

Comment:
This is not the same as before.

Reply:
@lunny Indeed, the first commit was correct, then in the second I inadvertently broke it (notice how previously `allowed` would be `true` if the Doer was neither the user to publicize nor an admin). This actually fixes that error. Feel free to check the logic, and that it's now correct, yourself.